scripts/docker/large/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+services:
+  main:
+    container_name: exec_large_test
+    image: dlewandowska/alpine-python-mock-file-publisher:latest
```

Comment:
image name suggests that it contains mock-file publisher. If it's true, what's the reason for including publisher in large tests for collector?